website/client/src/assets/css/sprites.css

```diff
@@ -190,6 +190,19 @@
   background: url("https://habitica-assets.s3.amazonaws.com/mobileApp/images/BackerOnly-Mount-Body-Gryphatrice.gif") no-repeat;
 }
 
+.Mount_Head_Dragon-Hydra, .Mount_Body_Dragon-Hydra {
+  width: 135px;
+  height: 135px;
```

Comment:
Since these are exactly the same property values as for the Gryphatrice, we can compact a lil bit and add these classes to line 180 instead of making a new declaration~